packages/central-server/src/apiV2/dataTables/assertDataTablePermissions.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { hasBESAdminAccess } from '../../permissions';
+
+const getPermissionListWithWildcard = async accessPolicy => {
+  const userPermissionGroups = accessPolicy.getPermissionGroups();
+  return ['*', ...userPermissionGroups];
+};
```

Comment:
This code isn't the DRY-est... it's copy pasted from dataElements permissions stuff. It's a bit awkward to consolidate, but I could make more of an effort if the reviewer feels it's worth it

Reply:
Sure no DRY is fine here